portsyncd/linksync.cpp

```diff
@@ -197,17 +159,6 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
                        nlmsg_type, key.c_str(), admin, oper, addrStr, ifindex, master);
     }
 
-    if (!key.compare(0, MGMT_PREFIX.length(), MGMT_PREFIX))
```

Comment:
I think we should remove the check at line 134 as well

Reply:
Removed this : key.compare(0, MGMT_PREFIX.length(), MGMT_PREFIX)